packages/custom_lint/test/cli_test.dart

```diff
@@ -11,13 +11,22 @@ final oyPluginSource = createPluginSource([
   TestLintRule(
     code: 'oy',
     message: 'Oy',
+    errorSeverity: 'ErrorSeverity.WARNING',
   )
 ]);
 
 final helloWordPluginSource = createPluginSource([
   TestLintRule(
     code: 'hello_world',
     message: 'Hello world',
+    errorSeverity: 'ErrorSeverity.WARNING',
```

Comment:
This impacts existing tests.

You should instead make a separate variable with the warning flag instead of reusing something already used by different tests but with the info status.

Reply:
before considering setting up another test, I would like to confirm [this discussion](https://github.com/invertase/dart_custom_lint/pull/129#discussion_r1233210398) first.